app/experimenter/nimbus-ui/src/hooks/useCommonForm/index.test.tsx

```diff
@@ -154,7 +154,7 @@ describe("hooks/useCommonForm", () => {
     it("FormAudience", () => {
       render(<AudienceSubject />);
       const filteredAudienceFieldNames = audienceFieldNames.filter(
-        (e) => e !== "languages",
+        (e) => e !== "languages" && e !== "populationPercent",
```

Comment:
I am curious to know, why this check??

Reply:
We had originally removed the use of these hooks from the population percent field. Since that's added back, this can be removed 👍